operators/olm_v1/index.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="olmv1-about"]
+= About Operator Lifecycle Manager v1 (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olmv1-about
+
+toc::[]
+
+{product-title} 4.14 introduces components for a next-generation iteration of Operator Lifecycle Manager (OLM) as a Technology Preview feature. Known during this phase as OLM v1, the updated framework evolves many of the concepts that have been part of the version of OLM included since the release of {product-title} 4.
+
+:FeatureName: OLM v1
+include::snippets/technology-preview.adoc[]
+
+During this Technology Preview phase of OLM v1 in {product-title} 4.14, administrators can use file-based catalogs to install and manage the following:
+
+* OLM-based Operators, similar to the existing OLM experience
+* Plain bundles, which are static collections of arbitrary Kubernetes manifests
+
+[id="olmv1-about-purpose"]
+== Purpose
+
+The mission of Operator Lifecycle Manager (OLM) has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, running, and updating functional extensions to the cluster easy, safe, and reproducible for cluster administrators and platform-as-a-service (PaaS) administrators, throughout the lifecycle of the underlying cluster.
+
+The existing version of OLM, which launched with {product-title} 4 and is included by default, was focused on providing unique support for these specific needs for a particular type of cluster extension, which have been referred as Operators. Operators are classified as one or more Kubernetes controllers, shipping with one or more API extensions (`CustomResourceDefinition` objects) to provide additional functionality to the cluster.
+
+After running OLM in production clusters for many releases, it became apparent that there is a desire to deviate from this coupling of CRDs and controllers to encompass the lifecycling of extensions that are not just Operators.
```

Comment:
I think you can get away with a noun here to avoid a new term
```suggestion
After running OLM in production clusters for many releases, it became apparent that there is a desire to deviate from this coupling of CRDs and controllers to encompass the lifecycles of extensions that are not just Operators.
```

Reply:
I wonder about this part more generally. It reads like something in an enhancement proposal (which is of course not inherently a bad thing), but seems a little off from the tone prod docs usually has. Could there be a more neutral problem statement that feels less narrative? I don't understand the details well enough to give a concrete suggestion, and like I said there is nothing wrong with it as-is, so just presented for your consideration :bow: